articles/connect-end-user-to-wifi-with-certificate.md

```diff
@@ -113,7 +123,9 @@ When saving the configuration, Fleet will attempt to connect to the SCEP server
 
 ### Step 2: Add SCEP configuration profile to Fleet
 
-1. Create a [configuration profile](https://fleetdm.com/guides/custom-os-settings) with the SCEP payload. In the profile, for `Challenge`, use`$FLEET_VAR_NDES_SCEP_CHALLENGE`. For `URL`, use `$FLEET_VAR_NDES_SCEP_PROXY_URL`.
+1. Create a [configuration profile](https://fleetdm.com/guides/custom-os-settings) with the SCEP payload. In the profile, for `Challenge`, use`$FLEET_VAR_NDES_SCEP_CHALLENGE`. For `URL`, use `$FLEET_VAR_NDES_SCEP_PROXY_URL`, and make sure to add `$FLEET_VAR_SCEP_RENEWAL_ID` to `CN`.
+
+> If you previously added a profile without `$FLEET_VAR_SCEP_RENEWAL_ID` it will work, but automati renewal won't be enabled.
```

Comment:
```suggestion
```

I think we can get away without documenting this. Why? DigiCert is experimental. We want all IT admins to use automatic renewal.